backend/analytics_server/mhq/api/pull_requests.py

```diff
@@ -167,3 +167,22 @@ def get_team_lead_time_trends(
         week.isoformat(): adapt_lead_time_metrics(average_lead_time_metrics)
         for week, average_lead_time_metrics in weekly_lead_time_metrics_avg_map.items()
     }
+
+
+@app.route("/teams/<team_id>/prs/merged_without_review", methods={"GET"})
+@queryschema(
+    Schema(
+        {
+            Required("from_time"): All(str, Coerce(datetime.fromisoformat)),
+            Required("to_time"): All(str, Coerce(datetime.fromisoformat)),
+        }
+    ),
+)
+def merged_without_review(team_id: str, from_time: datetime, to_time: datetime):
+    query_validator = get_query_validator()
+    team: Team = query_validator.team_validator(team_id)
+    interval: Interval = query_validator.interval_validator(from_time, to_time)
+    pr_analytics = get_pr_analytics_service()
+    result = pr_analytics.get_prs_merged_without_review(team.id, interval)
+    prs_map = [pr.id for pr in result]
+    return {"PrsWithoutReviewMerged": prs_map}
```

Comment:
I tested out this API. It just returns a list of ids. This is not useful for the front end. We need to send a list of PRs as data to the backend. We are already using a func in the same file in /lead_time/prs API. Please refer the implementation for that API. Should help.
![image](https://github.com/user-attachments/assets/e38d12f4-bdc2-41e4-8e55-047abd37a3a9)

Reply:
We don't name our property keys that go in API responses in that case either. @RISHIKESHk07